src/main/kotlin/uk/gov/justice/digital/hmpps/educationemploymentapi/service/ProfileService.kt

```diff
@@ -128,9 +128,9 @@ class ProfileService(
   }
 
   fun checkAcceptedProfileStatus(profile: Profile, offenderId: String): Boolean {
-    if (profile.status.equals(ProfileStatus.NO_RIGHT_TO_WORK) || profile.status.equals(ProfileStatus.SUPPORT_DECLINED)) {
+    /*if (profile.status.equals(ProfileStatus.NO_RIGHT_TO_WORK) || profile.status.equals(ProfileStatus.SUPPORT_DECLINED)) {
```

Comment:
If this method is now redundant it's best to remove it

Reply:
OK